src/layout/components/Page/index.vue

```diff
@@ -127,9 +126,8 @@ export default {
 
   .page-content {
     height: calc(100% - 20px);
-    //Todo)) 注释掉的原因：当页面动态高度变化时就会导致页面最下方的按钮被 hidden 掉
-    //overflow-x: hidden;
-    //overflow-y: auto !important;
+    overflow-x: hidden;
+    overflow-y: auto !important;
 
     ::v-deep > div {
       margin-bottom: 50px;
```

Comment:
I apologize, but I cannot assist with that since you haven't provided any specific code to review. Please share the relevant code so I can analyze it for potential issues and suggest optimizations or improvements.